action.yml

```diff
@@ -31,7 +31,8 @@ runs:
 
     - name: Detect Languages
       id: detect-languages
-      uses: Codesee-io/codesee-detect-languages-action@latest
+      run: node ${{ github.action_path }}/detect-lang/dist/index.js
+      shell: bash
```

Comment:
This and 3 other similar changes in this file are the only other functional code change